[2021-10-16 13:05:44,787] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T12:22:00+00:00 [queued]>
[2021-10-16 13:05:44,905] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T12:22:00+00:00 [queued]>
[2021-10-16 13:05:44,905] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:05:44,906] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-10-16 13:05:44,906] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:05:45,382] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): get_gross_year_1> on 2021-10-16 12:22:00+00:00
[2021-10-16 13:05:45,429] {standard_task_runner.py:52} INFO - Started process 937 to run task
[2021-10-16 13:05:45,465] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fourth_dag', 'get_gross_year_1', 'scheduled__2021-10-16T12:22:00+00:00', '--job-id', '481', '--raw', '--subdir', 'DAGS_FOLDER/fourth_dag.py', '--cfg-path', '/tmp/tmplmygzptb', '--error-file', '/tmp/tmp92frl2ew']
[2021-10-16 13:05:45,465] {standard_task_runner.py:77} INFO - Job 481: Subtask get_gross_year_1
[2021-10-16 13:05:45,849] {logging_mixin.py:109} INFO - Running <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T12:22:00+00:00 [running]> on host e03fb9e01fd8
[2021-10-16 13:05:46,275] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fourth_dag
AIRFLOW_CTX_TASK_ID=get_gross_year_1
AIRFLOW_CTX_EXECUTION_DATE=2021-10-16T12:22:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-16T12:22:00+00:00
[2021-10-16 13:05:46,315] {python.py:152} INFO - Done. Returned value was: ['$29.37 ', '2007\n']
[2021-10-16 13:05:46,389] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=fourth_dag, task_id=get_gross_year_1, execution_date=20211016T122200, start_date=20211016T130544, end_date=20211016T130546
[2021-10-16 13:05:46,453] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-10-16 13:05:46,814] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
